js/controllers/list/submissions/SubmissionsListItem.vue

```diff
@@ -37,10 +41,10 @@
 		<div v-else class="pkpListPanelItem--submission__stage">
 			<div class="pkpListPanelItem--submission__stageRow">
 				<div class="pkpListPanelItem--submission__stageLabel">
-					<template v-if="submission.status.id === 3 || submission.status.id === 4">
-						{{ submission.status.label }}
+					<template v-if="item.status.id === 3 || item.status.id === 4">
```

Comment:
Yeah, I've been juggling a few different ideas for getting constants out to the JS (see [this comment](https://github.com/pkp/ojs/pull/1337#issuecomment-306202130)). Can we file a new issue to handle this everywhere or would you like to see it addressed as part of this pull request?